### DISCUSSION TITLE
Enable `allow_sending_without_reply` by default.

### DISCUSSION THREAD

**Discussion by byehack** [eternnoir/pyTelegramBotAPI#1708] [Ideas]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.7
2. What OS are you using?
win11
3. What version of python are you using?
3.7

Example:
```python3
from telebot.async_telebot import AsyncTeleBot
bot = AsyncTeleBot('TOKEN', allow_sending_without_reply=True)
```

**Comment by coder2020official** [comment]
Good suggestion, but I already have this and other parameters in my backlog list

**Comment by coder2020official** [comment]
Btw, next time you can use discussions for suggestions

**Comment by coder2020official** [comment]
Added it in #1735
